src/stylesheets/common/_colors.scss

```diff
@@ -57,46 +57,75 @@ $okColor:#63a240;
 
 
 //Classes for styleguide
```

Comment:
Cool, thanks.  Yes, move to styleguidepage css and we can readdress naming scheme if we decide they should go elsewhere.